database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -12,12 +12,15 @@ class CreateUsersTable extends Migration {
 	 */
 	public function up()
 	{
-		Schema::create('users', function(Blueprint $table)
-		{
+		Schema::create('users', function (Blueprint $table) {
 			$table->increments('id');
+
+			$table->string('github_id')->unique();
 			$table->string('name');
-			$table->string('email')->unique();
-			$table->string('password', 60);
+			$table->string('email');
```

Comment:
I didn't have the "user:email" scope set up in the AuthController is the reason it wasn't bringing back the email. Fixed that. It's not completely necessary that we know it, but if you ever wanted to integrate giscus with gistlog it could be helpful.